grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java

```diff
@@ -263,11 +256,12 @@ void sendQueriesToLoader(Collection<Query> queries){
         Retryer<TaskResult> sendQueryRetry = RetryerBuilder.<TaskResult>newBuilder()
                 .retryIfExceptionOfType(IOException.class)
                 .retryIfRuntimeException()
+                .retryIfResult(r -> r != null && r.getCode().startsWith("5"))
```

Comment:
I'm fixing this awfulness next. I'll try to put a benchmark on REST and change the http client to something that doesn't require us to do the retry explicitly

Reply:
thanks for apologising